### PR TITLE
Add scraper for crashpadseries.com

### PR DESCRIPTION
**Pull request by jackchack**
Ran the validations, all passed

### CODE REVIEW

File: scrapers/CrashpadSeries.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: CrashpadSeries 
+debug:
+  printHTML: true
```
Comment (JaseNZC) [inline scrapers/CrashpadSeries.yml:3]: Don't think it needs debug still there.

File: scrapers/CrashpadSeries.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://crashpadseries.com
```
Comment (bnkai) [inline scrapers/CrashpadSeries.yml:7]: If there is not specific reason we usually do it like this
```suggestion
      - crashpadseries.com
```

File: scrapers/CrashpadSeries.yml (hunk 1 above)
Comment (jackchack) [reply scrapers/CrashpadSeries.yml]: woops! removed

File: scrapers/CrashpadSeries.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          - replace:
+            - regex: .+(?:"datePublished":")([^T]+).+
+              with: $1
+          - parseDate: 2006-01-02
+      Performers:
+        Name: //div[@id="primary-container"]//div[contains(@class,"model-thumb")]//h4
+      Tags:
+        Name: //div[@id="primary-container"]//a[contains(@class,"content-icon")]/@data-original-title
+      Details:
+        selector: //div[@id="primary-container"]//div[@class="ep-description"]
```
[the remark this answers is not in the document]
Comment (jackchack) [reply scrapers/CrashpadSeries.yml]: agree, looks like there is a span of tags I missed, fixed

File: scrapers/CrashpadSeries.yml (hunk 2 above)
Comment (jackchack) [reply scrapers/CrashpadSeries.yml]: fixed

**Comment by jackchack** [comment]
@bnkai @JaseNZC thanks for the comments, I have resolved them

**Pull request closed without merging**